metrics/metrics-core/src/main/java/io/servicecomb/metrics/core/event/InvocationStartProcessingEventListener.java

```diff
@@ -41,9 +41,14 @@ public InvocationStartProcessingEventListener(RegistryMonitor registryMonitor) {
   public void process(Event data) {
     InvocationStartProcessingEvent event = (InvocationStartProcessingEvent) data;
     InvocationMonitor monitor = registryMonitor.getInvocationMonitor(event.getOperationName());
+    //TODO:current java chassis unable know invocation type before starting process,so all type WaitInQueue increment(-1) (decrement)
```

Comment:
why?
event.getInvocationType is not you want?

Reply:
and in fact, we only trigger InvocationStartProcessingEvent  in producer invocation
missed it in consumer invocation.

Reply:
Great,it's a important info for me ,thank a lot

Reply:
Done,Consumer Side Event will be trigger in VertxHttpClient and HighwayClient,https://issues.apache.org/jira/browse/SCB-126